### DISCUSSION TITLE
设置也的推特链接错误

### DISCUSSION THREAD

**Discussion by zuixiangsi** [immersive-translate/immersive-translate#1282] [Help]
### 概述 | Summary

![绝对链接](https://github.com/immersive-translate/immersive-translate/assets/85442664/532dc81a-4831-4660-9791-271b04ea1859)
推特这里的链接加上“https://”吧，否则就会变成“chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/twitter.com/immersivetran”

**Comment by theowenyoung** [comment]
感谢反馈，下个版本修复～
